src/js/modules/core/components/HomePage.js

```diff
@@ -42,9 +88,11 @@ const styles = {
 
 //The filler column should have a borderRight. Wait until there is something there first
 
+/*
```

Comment:
Why is this code commented out. If you're not using it, delete it. You can use git to get it back if you want